tox.ini

```diff
@@ -17,51 +17,51 @@ whitelist_externals =
-    pytest -n 2 -m 'not es_tests and not needs_locales_compilation' -v src/olympia/devhub/ {posargs}
+    make -f Makefile-docker install_python_test_dependencies install_node_dependencies
+    pytest -n 2 -m 'not es_tests and not needs_locales_compilation and not static_assets' -v src/olympia/devhub/ {posargs}
 
 [testenv:reviewers]
 commands =
-    make -f Makefile-docker update_deps
-    pytest -n 2 -m 'not es_tests and not needs_locales_compilation' -v src/olympia/reviewers/ {posargs}
+    make -f Makefile-docker install_python_test_dependencies
+    pytest -n 2 -m 'not es_tests and not needs_locales_compilation and not static_assets' -v src/olympia/reviewers/ {posargs}
 
 [testenv:amo-locales-and-signing]
 commands =
-    make -f Makefile-docker update_deps
-    pytest -n 2 -m 'not es_tests and not needs_locales_compilation' -v src/olympia/amo/ src/olympia/lib/crypto/ src/olympia/signing {posargs}
+    make -f Makefile-docker install_python_test_dependencies install_node_dependencies
+    pytest -n 2 -m 'not es_tests and not needs_locales_compilation and not static_assets' -v src/olympia/amo/ src/olympia/lib/crypto/ src/olympia/signing {posargs}
     bash {toxinidir}/locale/compile-mo.sh {toxinidir}/locale/
     pytest -n 2 -m 'needs_locales_compilation' -v src/olympia/ {posargs}
 
 [testenv:users-and-accounts]
 commands =
-    make -f Makefile-docker update_deps
-    pytest -n 2 -m 'not es_tests and not needs_locales_compilation' -v src/olympia/users/ src/olympia/accounts/ {posargs}
+    make -f Makefile-docker install_python_test_dependencies
+    pytest -n 2 -m 'not es_tests and not needs_locales_compilation and not static_assets' -v src/olympia/users/ src/olympia/accounts/ {posargs}
 
 [testenv:main]
 commands =
-    make -f Makefile-docker update_deps
-    pytest -n 2 -m 'not es_tests and not needs_locales_compilation' -v src/olympia/ --ignore src/olympia/addons/ --ignore src/olympia/devhub/ --ignore src/olympia/reviewers/ --ignore src/olympia/amo/ --ignore src/olympia/users/ --ignore src/olympia/accounts/ --ignore src/olympia/lib/crypto --ignore src/olympia/signing {posargs}
+    make -f Makefile-docker install_python_test_dependencies install_node_dependencies
+    pytest -n 2 -m 'not es_tests and not needs_locales_compilation and not static_assets' -v src/olympia/ --ignore src/olympia/addons/ --ignore src/olympia/devhub/ --ignore src/olympia/reviewers/ --ignore src/olympia/amo/ --ignore src/olympia/users/ --ignore src/olympia/accounts/ --ignore src/olympia/lib/crypto --ignore src/olympia/signing {posargs}
 
 [testenv:ui-tests]
 commands =
     make -f Makefile-docker update_deps
     make -f Makefile-docker ui-tests
-    pytest --driver Firefox tests/ui/ {posargs}
+    pytest --driver Firefox tests/ui/ -k test_devhub_addon_upload {posargs}
 
 [testenv:assets]
 commands =
     make -f Makefile-docker update_deps
-    make -f Makefile-docker update_assets
+    pytest -m "static_assets" --ignore=tests/ui/ -v src/olympia/ {posargs}
```

Comment:
We are running `update_assets` after running `pytest` here on master. Can you add that back please?

Reply:
gah okay. I think atom saved old changes while I was rebasing.